src/react/components/styleform/AreaTab.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { SvgRadioButton } from 'oskari-ui';
+import { Form, Row } from 'antd';
+
+export const AreaTab = (props) => {
+    return (
+        <>
+            <Row>
+                <Form.Item name='lineStyle' label='Line dash' { ...props.formLayout }>
```

Comment:
Why does this have name and the Fill style one doesn't? Does it matter/affect something?

Reply:
Fix'd. Name is used for populating.